verification/verify_leaderboard_v2.py

```
@@ -0,0 +1,162 @@
+
+            print("Checking View Switcher...")
+            await page.wait_for_selector("button:has-text('GLOBAL WEALTH')", timeout=5000)
+            await page.wait_for_selector("button:has-text('SONG SCORES')", timeout=5000)
+
+            await page.screenshot(path="verification/success_leaderboard.png")
+            print("Success! Screenshot saved to verification/success_leaderboard.png")
+
+        except Exception as e:
+            print(f"Error: {e}")
+            await page.screenshot(path="verification/error_state_v2.png")
+            raise e
```

Ruff (0.15.2)

[warning] Use `raise` without specifying exception name

Remove exception name

(TRY201)